### DISCUSSION TITLE
Facial Recognition parameters tuning for twin brothers!

### DISCUSSION THREAD

**Discussion by benooye** [immich-app/immich#4150] [Q&A] [answered]
Thanks a lot for this amazing project!

I have tried Facial Recognition feature, but unfortunately there are a lot of mistakes with my twin brothers who is tagged as me!
How to tune the facial recognition parameters to better differenciate between me and my brother?
I see the MAX RECOGNITION DISTANCE parameter which is at 0.6 but I have no clue of the sensitivity of this parameter: Is 0.5 good? 0.2?

Thanks a lot in advance!

**Comment by bo0tzz** [comment, marked as answer]
See #3946

**Comment by benooye** [comment]
Thanks sorry for the duplicate
